### PR TITLE
HdfsBrowser: compatibility with Lab 4

### PR DESCRIPTION
**Pull request by diocas**
Kept the old build configuration to build both lab and nbextension as independent code

### CODE REVIEW

File: HdfsBrowser/pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.4.0", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
+requires = ["jupyter_packaging~=0.4.0", "jupyterlab>=4.0.0,<5", "setuptools>=40.8.0", "wheel"]
```
Comment (etejedor) [inline HdfsBrowser/pyproject.toml:2]: So JupyterLab is required to build?

File: HdfsBrowser/setup.py [hunk 2]
```diff
@@ -66,7 +66,7 @@
     cmdclass= cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab~=3.0",
```
Comment (etejedor) [inline HdfsBrowser/setup.py:69]: But not required to install?

File: HdfsBrowser/pyproject.toml (hunk 1 above)
Comment (diocas) [reply HdfsBrowser/pyproject.toml:2]: Yes.

File: HdfsBrowser/setup.py (hunk 2 above)
Comment (diocas) [reply HdfsBrowser/setup.py:69]: Missing.

**Review by etejedor — APPROVED** [review]

**Review by PMax5 — APPROVED** [review]

File: HdfsBrowser/package.json [hunk 3]
```diff
@@ -13,11 +13,15 @@
   "bugs": {
     "url": "https://github.com/swan-cern/jupyter-extensions/issues"
   },
+  "author": {
+    "name": "SWAN Team",
+    "email": ""
```
[the remark this answers is not in the document]
Comment (diocas) [reply HdfsBrowser/package.json:18]: The tool that automatically migrates the lab extensions requires it, otherwise it crashes.

**Pull request merged**